lib/zendesk_apps_support/validations/translations.rb

```diff
@@ -17,6 +17,7 @@ def call(package)
             path_match = TRANSLATIONS_PATH.match(file.relative_path)
             if path_match
               errors << locale_error(file, path_match[1]) << json_error(file)
+              errors << required_keys(file) if errors.compact.empty?
```

Comment:
Before this there was more duplication in `required_keys` of the guards that are in `json_error`. This is one way to avoid needing to check twice.